tests/conftest.py

```diff
@@ -260,20 +261,41 @@ def pytest_addoption(parser):
     parser.addoption('--test_downloads', action='store_true', default=False)
 
 
-def marker_names(item):
+def pytest_configure(config: pytest.Config):
+    """Add filterwarnings for vtk < 9.1 and numpy bool deprecation"""
+    warnings = config.getini('filterwarnings')
+
+    if pyvista.vtk_version_info < (9, 1):
+        warnings.append(
+            r'ignore:.*np\.bool.{1} is a deprecated alias for the builtin .{1}bool.*:DeprecationWarning'
+        )
+    if any(Path(a).stem == 'tst_doc_images' for a in config.args):
+        warnings.append(
+            'ignore:Unknown config option.*image_cache_dir.*:pytest.PytestConfigWarning'
+        )
```

Comment:
Added here because the warning is emitted during test collection, ie. before `pytest_collection_modifyitems`